config/testgrids/conformance/kind/conformance-kind.yaml

```diff
@@ -1,2 +0,0 @@
-dashboards:
-- name: conformance-kind
```

Comment:
this should really be a seperate commit and/or PR

Reply:
I split it to 2 commits.

Reply:
/hold cancel
Yes, doing some cleanup